Chp8Lsn4/Chp8Lsn4/Program.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chp8Lsn4
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            int convertNumber = UserInput();
+            int[] binaryNumber = DecimalToBinary(convertNumber);
```

Comment:
This looks broken.  You are calling `DecimalToBinary`, but there is no method with that name.  It seems like something did not 'commit' and/or 'push' correctly?